integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/KafkaSteps.java

```diff
@@ -0,0 +1,50 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.resources.kafka.KafkaResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class KafkaSteps {
+
+    private TestContext context;
+
+    public KafkaSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @Given("^Kafka bootstrap server is set$")
```

Comment:
Thinking what is the purpose of this step.
Should it skip the test in case bootstrap server is not defined? Do we actually have such cases?